src/test/java/api/support/builders/ItemRequestBuilder.java

```diff
@@ -29,13 +29,14 @@
   private final String itemLevelCallNumberPrefix;
   private final String itemLevelCallNumberSuffix;
   private final String hrid;
+  private final String copyNumber;
 
   public ItemRequestBuilder() {
     this(UUID.randomUUID(), null, null, null, "645398607547",
       AVAILABLE_STATUS, bookMaterialType(), null, null, null,
       canCirculateLoanType(), null, null, null,
       null, null, null,
-      null
+      null, null
```

Comment:
Given that multiple values are present on the same line, these null values can be included on the previous line.